lua/treesitter-context/context.lua

```diff
@@ -215,6 +215,11 @@ local function get_parent_langtrees(bufnr, range)
     return {}
   end
 
+  if vim.version.ge(vim.version(), {0, 11, 0}) then
+    root_tree:parse(range, function(...) end)
+  else
+    root_tree:parse(range)
+  end
```

Comment:
This won't work. You just need to include the correct `@diagnostic: disable-next-line` annotation.